### PR TITLE
Onboarding

### PR DESCRIPTION
**Pull request by ofagbemi**
Add UI for onboarding. Remaining todo
- Store whether someone's closed the card before with CoreData or something (idk how, so looking at you @andrewfang). I'll explain how to in the comments.

### CODE REVIEW

File: ios/rüm/DataViewController.swift [hunk 1]
```diff
@@ -165,17 +168,25 @@ class DataViewController: UIViewController, UICollectionViewDelegateFlowLayout,
         }
     }
     
+    // MARK: - card delegation
+    func userDidCloseCardView(cardView: CardViewController) {
```
Comment (ofagbemi) [inline ios/rüm/DataViewController.swift:177]: Add the code to store whether the user's closed the Kudos onboarding card before here

File: ios/rüm/MainViewController.swift [hunk 2]
```diff
@@ -399,17 +400,30 @@ class MainViewController: UIViewController, UITableViewDelegate, UITableViewData
...
+                    CardContent(header: "Add a task", content: "You can add tasks to the todo list by hitting the “+” button at the bottom of the screen."),
+                    CardContent(header: "Cross it off", content: "When you’ve completed a task, just swipe it away to let everyone know you’re done."),
+                    CardContent(header: "Do it again", content: "You can use the “I just…” section to quickly add and check off common tasks.")
+                ]
+                cardVC.delegate = self
+            }
         }
     }
     
+    func userDidCloseCardView(cardView: CardViewController) {
```
Comment (ofagbemi) [inline ios/rüm/MainViewController.swift:427]: Add the code to store whether the user's closed the Todo onboarding card before here

**Comment by ofagbemi** [comment]
Check if the user's closed the Todo onboarding card before in the Todo page's `viewDidLoad` and just remove the container view (`self.onboardingCardContainerView`) from the super view.

Check if the user's closed the Kudos onboarding card before in the Kudos page's `viewDidLoad` and just remove the container view (`self.onboardingCardContainerView`) from the super view.

File: ios/rüm/MainViewController.swift (hunk 2 above)
Comment (andrewfang) [inline ios/rüm/MainViewController.swift:427]: yep done

File: ios/rüm/DataViewController.swift (hunk 1 above)
Comment (andrewfang) [inline ios/rüm/DataViewController.swift:177]: also done

**Comment by andrewfang** [comment]
Looks good. I added all the stuff. Merge when ready.

**Pull request merged**